tools/cli/test/data/changelog/same-api-version/base/openapi-2023-01-01.json

```diff
@@ -10495,7 +10495,7 @@
         } ],
         "summary" : "Return All Project Invitations",
         "tags" : [ "Projects" ],
-        "x-sunset" : "2024-10-04"
+        "x-sunset" : "2099-10-04"
```

Comment:
> e2e tests should not depend on our current changelog right? we can address in the TODO ticket?

@blva 
The e2e test currently uses specs from MMS that have the sunset date set to 2024-10-04 (this is the change included in my PR). I’ll create a follow-up PR to update the e2e test specs to set the endpoint sunset date to 2099, so that we won’t encounter this error again.

Reply:
ah ok, it is a "frozen" spec but we reached the sunset date. fun! xD